functions.php

```diff
@@ -339,9 +339,12 @@ function vf_get_link_url($options) {
 function vf_get_user() {
    global $current_user, $wp_roles;
 
-   if (!function_exists('get_currentuserinfo'))
+   if (!function_exists('get_currentuserinfo')) {
       require_once ABSPATH . WPINC . '/pluggable.php';
-   get_currentuserinfo();
+      get_currentuserinfo();
+   } else {
+       wp_get_current_user();
+   }
```

Comment:
This looks off.

Reply:
get_currentuserinfo() throws a warning in wordpress. For backwards compatibility I left in because it still works.